azext_iot/tests/test_device_digitaltwin_invoke_command.json

```diff
@@ -0,0 +1,5 @@
+{
+    "blinkRequest": {
```

Comment:
is this the payload for this command? 
      {
        "@type": "Command",
        "commandType": "synchronous",
        "description": "This command will begin blinking the LED for given time interval.",
        "name": "blink",
        "request": {
          "name": "blinkRequest",
          "schema": "long"
        },
        "response": {
          "name": "blinkResponse",
          "schema": "string"
        }
      }
If so, the payload should simply be 1. We fixed the bug in the UI, where we should not wrap the name "blinkRequest" in command payload. So you would need to remove "blinkRequest". But I am not sure where did this "interval" key come from.

Reply:
@anusapan , please address this -- my only concern being we have accurate test assets (payloads, interfaces, models etc)